### ISSUE TITLE
Per CTF guest list not updating automatically when new users are registered

### ISSUE THREAD

**Issue by peace-maker** [TFNS/CTFNote#346]
When new users are added while you have CTFNote open already, the Guest list is not updated. You have to reload the page to see the updated user list. The same thing is true for users getting deleted, they will still show up in the Guests view until you refresh the page.

![grafik](https://github.com/user-attachments/assets/8ca431b4-e03d-4e36-81e4-86a49ac6542d)

**Comment by JJ-8** [comment]
Yep, this is indeed the case since we do not announce an update over websockets for the CTF component where the guests are included. To fix this we need to push a whole new CTF state to all the connected clients to update the guests tab.
